android/capacitor/src/main/java/com/getcapacitor/BridgeWebViewClient.java

```diff
@@ -92,4 +93,19 @@ public void onReceivedHttpError(WebView view, WebResourceRequest request, WebRes
             view.loadUrl(errorPath);
         }
     }
+
+    @Override
+    public boolean onRenderProcessGone(WebView view, RenderProcessGoneDetail detail) {
+        super.onRenderProcessGone(view, detail);
+        List<WebViewListener> webViewListeners = bridge.getWebViewListeners();
+        if (webViewListeners != null) {
+            for (WebViewListener listener : bridge.getWebViewListeners()) {
+                if (listener.onRenderProcessGone(view, detail)) {
+                    return true;
```

Comment:
I think it should keep iterating to forward the event to all the listeners even if one of them returned true

Reply:
Just noting this feedback hasn't been addressed here yet.
The logical combination will short-circuit, so once a true value is returned, I don't believe any other listener gets informed.